pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -55,6 +55,86 @@ type KeycloakAPIRealm struct {
 	// TODO: change to values and use kubebuilder default annotation once supported
 	// +optional
 	AdminEventsDetailsEnabled *bool `json:"adminEventsDetailsEnabled,omitempty"`
+
+	// Authentication flows
+	// +optional
+	AuthenticationFlows *[]KeycloakAPIAuthenticationFlow `json:"authenticationFlows,omitempty"`
```

Comment:
There seems to be *two* naming conventions here:

* `KeycloakAPI` + Java class name - `Representation` (Example: KeycloakAPIRealm)
* `Keycloak` + Java class name - `Representation` (example: KeycloakClient)

This could be an inconvenience using code gen later.

Reply:
Those `KeycloakAPI*` objects correspond to Keycloak Server REST representations. To give a specific example here: `KeycloakAPIClient` should be more or less equal to [Client Representation](https://www.keycloak.org/docs-api/10.0/rest-api/index.html#_clientrepresentation).